presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/CassandraTestingUtils.java

```diff
@@ -220,6 +222,24 @@ public static void createTableAllTypesPartitionKey(CassandraSession session, Sch
         insertTestData(session, table, date, 9);
     }
 
+    public static void createTableTupleType(CassandraSession session, SchemaTableName table)
+    {
+        String typeName = "type_tuple_type";
+
+        session.execute("DROP TABLE IF EXISTS " + table);
+
+        session.execute("CREATE TABLE " + table + " (" +
+                " typeinteger int PRIMARY KEY, " +
+                " typetuple frozen<tuple<int, int>>" +
+                ")");
+        session.execute("INSERT INTO " + table + " (typeinteger, typetuple)" +
+                "VALUES (1, (1, 1))");
+        session.execute("INSERT INTO " + table + " (typeinteger, typetuple)" +
+                "VALUES (2, (2, 2))");
```

Comment:
No need to separate to 2 lines since it isn't long. Also, let's use `String.format`.
```java
session.execute(String.format("INSERT INTO %s ... VALUES ...", table))
```
If a variable is in end of the line like `"SELECT COUNT(*) FROM " + table`, you don't need to use String.format.